tests/filecheck/transforms/dispatch_kernels.mlir

```diff
@@ -57,3 +57,55 @@
 //CHECK-NEXT:     "func.return"(%arg2) : (memref<?x128xi32>) -> ()
 //CHECK-NEXT:   }) : () -> ()
 //CHECK-NEXT: }) : () -> ()
+
+// -----
+
+"builtin.module"() ({
+  "func.func"() <{function_type = (memref<16xi64>, memref<16xi64>, memref<16xi64>) -> (), sym_name = "streamer_add", sym_visibility = "public"}> ({
+  ^bb0(%arg0: memref<16xi64>, %arg1: memref<16xi64>, %arg2: memref<16xi64>):
+    "linalg.generic"(%arg0, %arg1, %arg2) <{indexing_maps = [affine_map<(d0) -> (d0)>, affine_map<(d0) -> (d0)>, affine_map<(d0) -> (d0)>], iterator_types = [#linalg.iterator_type<parallel>], operandSegmentSizes = array<i32: 2, 1>}> ({
+    ^bb0(%arg3: i64, %arg4: i64, %arg5: i64):
+      %0 = "arith.addi"(%arg3, %arg4) : (i64, i64) -> i64
+      "linalg.yield"(%0) : (i64) -> ()
+    }) : (memref<16xi64>, memref<16xi64>, memref<16xi64>) -> ()
+    "func.return"() : () -> ()
+  }) : () -> ()
+}) : () -> ()
+
+// CHECK:  "builtin.module"() ({
+// CHECK-NEXT:    "func.func"() <{"function_type" = (memref<16xi64>, memref<16xi64>, memref<16xi64>) -> (), "sym_name" = "streamer_add", "sym_visibility" = "public"}> ({
+// CHECK-NEXT:    ^0(%arg0 : memref<16xi64>, %arg1 : memref<16xi64>, %arg2 : memref<16xi64>):
+// CHECK-NEXT:      "linalg.generic"(%arg0, %arg1, %arg2) <{"indexing_maps" = [affine_map<(d0) -> (d0)>, affine_map<(d0) -> (d0)>, affine_map<(d0) -> (d0)>], "iterator_types" = [#linalg.iterator_type<parallel>], "operandSegmentSizes" = array<i32: 2, 1>, "library_call" = "snax_alu_stream"}> ({
+// CHECK-NEXT:      ^1(%arg3 : i64, %arg4 : i64, %arg5 : i64):
+// CHECK-NEXT:        %0 = "arith.addi"(%arg3, %arg4) : (i64, i64) -> i64
+// CHECK-NEXT:        "linalg.yield"(%0) : (i64) -> ()
+// CHECK-NEXT:      }) : (memref<16xi64>, memref<16xi64>, memref<16xi64>) -> ()
+// CHECK-NEXT:      "func.return"() : () -> ()
+// CHECK-NEXT:    }) : () -> ()
+// CHECK-NEXT:  }) : () -> ()
```

Comment:
Maybe only check the library_call attribute? Also, maybe you can just check if it is _not_ set to snax_alu_stream instead?

Reply:
yes, because it checks for "snax_alu", the quotations are checked as well, so snax_alu_stream is impossible